helpers/sortAndFilterHotels.js

```diff
@@ -0,0 +1,55 @@
+/* jslint node: true */
+'use strict';
+
+var _ = require('lodash');
+var sortBy = require('../lib/sortBy');
+
+module.exports = function(dust) {
+
+  /*
+   * @disclaimer - this is a temporary filter helper to separate a recommended hotel(s) from a list of hotels. This is temporary because it's written in dust, which will change soon.
+   *
+   * @description Augmented sorting method that orders a list of things based on a child node, and then filters the result based on an object and a value .
+   * @param {object} parent Object to sort
+   * @param {string} node Child node to sort object by
+   * @param {object} filterArray - array to base the filter on
+   * @param {boolean as string} exclude - if 'true', the filter function will exclude all items in the filterArray, based on the filterValue
+   * @param {boolean as string} include - if 'true', the filter function will include all items in the filterArray
+   * @example {@_sortAndFilterHotels parent=packageRatesReply.packageRates node="grossPrice" filterArray=_brandConfig.featuredHotel filterValue="x.links.hotelProducts.ids[0]" include="true"} {/_sortAndFilterHotels} output loop of parent object sorted by node and filtered by filterArray
+   */
+
+  dust.helpers._sortAndFilterHotels = function(chunk, context, bodies, params) {
+    var parent = params.parent;
+    var node = params.node;
```

Comment:
As both of these variables are only used once and the name isn't really adding more context, this isn't worth declaring. You may as well reference directly on L28